project_testing/housing_characteristics/HVAC Cooling Efficiency.tsv

```diff
@@ -1,18 +1,3 @@
-Dependency=HVAC Cooling Type	Dependency=HVAC Has Shared System	Option=AC, SEER 8	Option=AC, SEER 10	Option=AC, SEER 13	Option=AC, SEER 15	Option=Heat Pump	Option=None	Option=Room AC, EER 8.5	Option=Room AC, EER 9.8	Option=Room AC, EER 10.7	Option=Room AC, EER 12.0	Option=Shared Cooling
```

Comment:
I don't know if these dependencies are needed in `project_testing`. May need to test.

Reply:
Integrity checks pass without these dependencies.